profile/README.md

```diff
@@ -30,7 +30,93 @@ Data about food is of public interest and has to be open. The complete database
 
 * <https://world.openfoodfacts.org/discover>
 
-## Priorities
+## How can I help?
+
+To start contributing, the easiest way it to join us on Slack <https://openfoodfacts.slack.com/> and post an introduction about, what you're interested in, and what you would like to do.
+This would allows other contributors to pin point projects that could match your interests. For example
+
+> I'm interested by the environemental impact of food and would like to help. I'm a designer, but I would also be interested by how you estimate the environmental impact of products.
+
+> I'm using the App, and wondering how could I help to improve it.
+
+> I'm a student, in computer science, and would like to help with some developpement. I already do some React, and Python.
+
+### Main roles
+
+Since we are on GitHub, you can guess that OpenFoodFacts needs some contributions from developpers and designers (main projects are detailed in next section). But their is plenty of other way to contribute. For example, you could:
```

Comment:
```suggestion
Since we are on GitHub, you can guess that Open Food Facts needs some contributions from developers and designers (main projects are detailed in next section). But there are plenty of other ways to contribute. For example, you could:
```